.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: java
 
+os:
+  - linux
+  - osx
+
 jdk:
+  - oraclejdk11
```

Comment:
Why do we test on another JDK 11?

Reply:
I think I read that the Implementations can be different so I added this. oraclejdk8 did not work for linux  (I think if a different distro is used than it is possible) and osx.
My guess is that the implementations are probably the same so I am ok with it if you want to remove it.

Reply:
OpenJDK 11 and Oracle JDK 11 are very similar and I'm willing to bet that they behave the same here, so let's remove this JDK to reduce build times.